packages/api-v2/src/hooks/useCFDCompareAccounts.tsx

```diff
@@ -136,7 +136,12 @@ const useCFDCompareAccounts = (isEU?: boolean) => {
     const demoAvailableAccounts = useMemo(() => {
         if (!sortedMt5Accounts) return;
         if (isEU) return sortedMt5Accounts.filter(account => account.shortcode === JURISDICTION.MALTAINVEST);
-        return sortedMt5Accounts.filter(account => account.shortcode === JURISDICTION.SVG);
+        return sortedMt5Accounts.filter(account => {
+            if (account.product === 'zero_spread') {
```

Comment:
@hasan-deriv, wouldn't it be  good idea to move this to a constant somewhere? I see it being used in quite a few places. 
```suggestion
            zero_spread
```